pkg/cmd/docgen/diagrams.go

```diff
@@ -583,6 +583,19 @@ var specs = []stmtSpec{
 			"relation_expr":          "table_name",
 		},
 	},
+	{
+		name:    "alter_table_partition_by",
+		stmt:    "alter_onetable_stmt",
+		inline:  []string{"opt_partition"},
```

Comment:
This doesn't appear to be working. Maybe because it's a replacement rule instead of a real production?